index.js

```diff
@@ -116,12 +116,93 @@ LocationForecast.prototype = {
       if (err) {
         return callback(err, null);
       }
-      
+
       return callback(null, res);
     });
   },
 
 
+    /**
+     * Get seven day weather.
+     * @param {Function} callback
+     */
+    getSevenDaySummary: function(callback) {
```

Comment:
Yep good idea. I was just quickly adding a feature so I could use it in my app. I didn't realise you could add a fork to package.json which is pretty handy. I'll implement this idea